modules/ROOT/pages/working-with-sandboxes.adoc

```diff
@@ -0,0 +1,266 @@
+       <members>ExchangeAssetIcons</members>
+       <members>ExchangeAssetIconsDevx</members>
+       <members>ExchangeAssetIconsEu</members>
+       <members>ExchangeAssetIconsQax</members>
+       <members>ExchangeAssetIconsStgx</members>
+       <members>GStaticFonts</members>
+       <members>GoogleFonts</members>
+       <members>SegmentAPI</members>
+       <members>SegmentCDN</members>
+       <name>CspTrustedSite</name>
+   </types>
+   <types>
+       <members>Anypoint</members>
+       <members>AnypointDevx</members>
+       <members>AnypointEu</members>
+       <members>AnypointQax</members>
+       <members>AnypointStgx</members>
+       <members>ExchangeAssetIcons</members>
+       <members>ExchangeAssetIconsDevx</members>
+       <members>ExchangeAssetIconsEu</members>
+       <members>ExchangeAssetIconsQax</members>
+       <members>ExchangeAssetIconsStgx</members>
+       <members>SegmentAPI</members>
+       <name>RemoteSiteSetting</name>
+   </types>
+   <!--
+   <types>
+       <members>ExperienceBundle</members>
+       <members>Communities</members>
+       <name>Settings</name>
+   </types>
+   -->
+   <types>
+       <members>API_Experience_Hub_Manage_Networks</members>
+       <name>PermissionSet</name>
+   </types>
+   <types>
+       <members>API Experience Hub</members>
+       <name>Network</name>
+   </types>
+   <types>
+       <members>HTTP_API_Audience</members>
+       <name>Audience</name>
+   </types>
+   <types>
+       <members>API_Experience_Hub</members>
+       <name>CustomSite</name>
+   </types>
+   <types>
+       <members>API_Experience_Hub1</members>
+       <name>ExperienceBundle</name>
+   </types>
+   <types>
+       <members>AEH_Default_Navigation</members>
+       <members>AEH_Default_User_Navigation</members>
+       <name>NavigationMenu</name>
+   </types>
+   <types>
+       <members>cbAPI_Experience_Hub</members>
+       <name>NetworkBranding</name>
+   </types>
+   <!--
+   <types>
+       <members>API Experience Hub Member User</members>
+       <name>Profile</name>
+   </types>
+   -->
+   <version>60.0</version>
+</Package>
+----
+====
+.. In a VCS, create a development branch.
+.. Download the metadata using the manifest created from the Salesforce sandbox organization. 
+. Retrieve the metadata from the Salesforce sandbox organization using Salesforce CLI commands:
+.. Log in to the Salesforce sandbox organization.
++
+[codeblock]
+----
+sf auth web login --instance-url=https://<SANBOX-INSTANCE_URL> --alias sandboxOrg
+----
+.. Edit the `manifest/package.xml` file before retrieving the metadata from the sandbox organization to retrieve the new metadata that you added.
++
+Your sandbox organization can contain a large amount of metadata from different products because it's a copy of the Salesforce production organization. API Experience Hub is a subset of it and it's specified in the `manifest/package.xml` file that was created when creating the Git repository in a previous step. 
```

Comment:
```suggestion
Your sandbox organization can contain metadata from multiple products because it's a copy of the Salesforce production organization. API Experience Hub is a subset of the sandbox organization specified in the `manifest/package.xml` file created when you created the Git repository in a previous step. 
```

Reply:
(Not sure if that changes the meaning.)